portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,57 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <meta charset="UTF-8" />
+    <title>Jonathan Chávez</title>
+    <link rel="stylesheet" href="style.css" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/typed.js/2.0.11/typed.min.js"></script>
+    <script src="script.js" defer></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div>
+      <header>
+        <h1 class="fadeIn-2">
+          <span class="highlight"
+            >This is Jonathan.<span class="blinks">|</span></span
+          >
```

Comment:
fyi, the line break seems a bit strange here and below for the img

Reply:
Solved it manually, it was something with the formatter.